govcd/vdc.go

```diff
@@ -23,13 +23,26 @@ type Vdc struct {
 	VApp   *types.VApp
 }
 
+type OrgVdcRecord struct {
+	OrgVdc *types.QueryResultOrgVdcRecordType
+	client *Client
+}
+
 func NewVdc(cli *Client) *Vdc {
 	return &Vdc{
 		Vdc:    new(types.Vdc),
 		client: cli,
 	}
 }
 
+// create instance with reference to types.OrgVdcRecord
+func NewOrgVdcRecord(cli *Client) *OrgVdcRecord {
```

Comment:
Same as above. Only entities that are handled through direct API calls need this treatment.